packages/drivers/iframe-driver/src/innerDocumentServiceFactory.ts

```diff
@@ -29,6 +29,7 @@ export class InnerDocumentServiceFactory implements IDocumentServiceFactory {
                 outerProxy = Comlink.wrap<IDocumentServiceFactoryProxy>(Comlink.windowEndpoint(window.parent));
             }
             if (outerProxy) {
+                console.log(outerProxy.connected.toString());
```

Comment:
Remove?

Reply:
or replace with call to logger

---
In reply to: [512853804](https://github.com/microsoft/FluidFramework/pull/4106#discussion_r512853804) [](ancestors = 512853804)